frontend/src/metabase/admin/settings/components/SettingsPages/GeneralSettingsPage.unit.spec.tsx

```diff
@@ -0,0 +1,163 @@
+import userEvent from "@testing-library/user-event";
+import fetchMock from "fetch-mock";
+
+import {
+  setupDashboardEndpoints,
+  setupPropertiesEndpoints,
+  setupSettingsEndpoints,
+  setupUpdateSettingEndpoint,
+} from "__support__/server-mocks";
+import { renderWithProviders, screen, waitFor } from "__support__/ui";
+import { UndoListing } from "metabase/containers/UndoListing";
+import type { SettingKey } from "metabase-types/api";
+import {
+  createMockDashboard,
+  createMockSettingDefinition,
+  createMockSettings,
+} from "metabase-types/api/mocks";
+
+import { GeneralSettingsPage } from "./GeneralSettingsPage";
+
+const generalSettings = {
+  "site-name": "Metabased",
+  "custom-homepage": true,
+  "custom-homepage-dashboard": 4242,
+  "dismissed-custom-dashboard-toast": false,
+  "site-url": "https://mysite.biz",
+  "admin-email": "help@mysite.biz",
+  "anon-tracking-enabled": false,
+  "redirect-all-requests-to-https": false,
+  "humanization-strategy": "simple",
+  "enable-xrays": false,
+  "allowed-iframe-hosts": "https://cooldashboards.limo",
+} as const;
+
+const setup = async () => {
+  const settings = createMockSettings(generalSettings);
+
+  fetchMock.get("https://mysite.biz/api/health", { status: 200 });
+
+  setupPropertiesEndpoints(settings);
+
+  setupDashboardEndpoints(
+    createMockDashboard({
+      id: 4242,
+      name: "My dashboard",
+    }),
+  );
+
+  setupUpdateSettingEndpoint();
+  setupSettingsEndpoints(
+    Object.entries(settings).map(([key, value]) =>
+      createMockSettingDefinition({ key: key as SettingKey, value }),
```

Comment:
I thought about doing this as well to avoid duplicating settings values. But I notice you didn't do this in some of the other test files. Is there a reason it's different in some cases?